huntsman/tests/test_camera.py

```diff
@@ -235,7 +236,7 @@ def test_exposure_scaling(camera, tmpdir):
     else:
         fits_path = str(tmpdir.join('test_exposure_scaling.fits'))
         camera.take_exposure(filename=fits_path, dark=True, blocking=True)
-        image_data, image_header = fits_utils.getdata(fits_path, header=True)
+        image_data, image_header = fits.getdata(fits_path, header=True)
```

Comment:
Oh, hm. I should support returning the header from the `fits_utils` although I'm kind of surprised this just doesn't work.

Reply:
Yeah, this was showing up in test failures. This test is usually skipped, but something broke the skipping while I was working on this PR and I found that `fits_utils` doesn't have `get_data` (at least not the one from POCS `develop`).